taskfiles/Terraform.yml

```diff
@@ -17,7 +17,6 @@ tasks:
   run-tests-against-submodule:
     cmds:
       - git submodule init {{.ROOT_DIR}}/{{.TEST_DIR}}/opslevel_modules
-      - git submodule update {{.ROOT_DIR}}/{{.TEST_DIR}}/opslevel_modules
```

Comment:
why is this being removed?  seems weird since the task's purpose is to run against the submodule?

Reply:
This is a huge inconvenience when testing out feature branches in `opslevel_modules`. But this update can come later and with a better solution too. I'll walk this back